src/core/agent/event.types.ts

```diff
@@ -32,6 +34,12 @@ interface OperationPendingEvent extends BaseEventEmitter {
     opType: OperationPendingRecordType;
   };
 }
+interface AddNewOperationPendingEvent extends BaseEventEmitter {
```

Comment:
As always, I will continue to ask for spacing between things - so a blank line above this :D

readability is important for maintainability